.nvmrc

```
@@ -1 +1 @@
-v18.14.0
+v20.9.0
```

```suggestion
v20.10.0
```
This is the latest version of v20 node